cmd/geth/snapshot.go

```diff
@@ -224,6 +225,8 @@ the expected order for the overlay tree migration.
 	}
 )
 
+const minBlockAmountReserved uint64 = 90000
```

Comment:
Maybe can reuse this const [FullImmutabilityThreshold](https://github.com/bnb-chain/bsc/blob/master/params/network_params.go#L71).